app.py

```diff
@@ -52,33 +51,36 @@ def run_app(self):
         (current_in, current_out) = self.get_current_frame_range(self.engine.name)
 
         if new_in is None or new_out is None:
-            message =  "Shotgun has not yet been populated with \n"
-            message += "in and out frame data for this Shot."
+            message = ("Shotgun has not yet been populated with \n"
+                       "in and out frame data for this Shot.")
```

Comment:
I know the way you did is better, but changing stuff like this will make it harder to later upgrade with a version coming from Shotgun. We need to avoid unnecessary changes as much as possible, when possible. The more changes we do, and the harder it is to merge Shotgun's master branch into ours.